src/components/crownConfig.vue

```diff
@@ -109,7 +111,7 @@ export default {
   computed: {
     style() {
       const { x, y, width: shapeWidth } = this.shape.findView(this.paper).getBBox();
-      const width = (this.node.diagram.bounds.width * this.paper.scale().sx);
+      const width = (this.shape.getBBox().width * this.paper.scale().sx) - 20;
```

Comment:
You removed some code I put in to get the trash icon closer to the sequence/message flows. Now it subtracts 40.
```suggestion
->      let width = (this.shape.getBBox().width * this.paper.scale().sx) - 20;
      const widthOffsetFromLabel = shapeWidth - width > 3 ? (shapeWidth / 2) - 15 : 0;
      if ([
        'processmaker-modeler-sequence-flow',
        'processmaker-modeler-message-flow',
      ].includes(this.node.type)) {
        width = shapeWidth;
      }
      
      const widthOffsetFromLabel = shapeWidth - width > 4 ? (shapeWidth / 2) - 15 : 0;
      return {
        top: `${y - 45}px`,
->        left: `${x + width + widthOffsetFromLabel}px`,
        cursor: 'pointer',
      };
```